rules/common.smk

```diff
@@ -12,6 +12,18 @@ def subtypes_by_subtype_wildcard(wildcards):
     }
     return(db[wildcards.subtype])
 
+
+def no_sign_request(_wildcards):
+    """
+    Returns the `--no-sign-request` option for AWS CLI if
+    the requested `S3_SRC` poins to the NEXTSTRAIN_PUBLIC_BUCKET.
+    """
+    NEXTSTRAIN_PUBLIC_BUCKET = "s3://nextstrain-data/"
```

Comment:
It would be more robust to call `aws s3api get-bucket-policy-status` to check if the bucket is public or not rather than relying on a hard-coded bucket here. 

I'm going to punt on that and say we can implement it in the [ingest/download-from-s3](https://github.com/nextstrain/ingest/blob/main/download-from-s3) script and update this workflow to call `download-from-s3` later.